packages/dialtone-vue3/components/tooltip/tooltip.vue

```diff
@@ -364,6 +370,8 @@ export default {
     this.externalAnchor && this.addExternalAnchorEventListeners();
     this.tip = createTippy(this.anchor, this.initOptions());
 
+    this.isTouchDevice = (('ontouchstart' in window) || (navigator.maxTouchPoints > 0) || (navigator.msMaxTouchPoints > 0));
```

Comment:
This is not a great way to detect touch as this only checks if they are using a touch enabled device. What if someone was on a laptop with a touchscreen but they were using the mouse? In this case tooltips would not be shown.

Instead lets add the "touchstart" event to the wrapper span element `@touchstart="onTouchStart"`. When this event is triggered it means the user has actually "touched" the element and in this case we should not show the tooltip. This should be much easier to test as well.

Reply:
@braddialpad I have made changes based on your comment, and it did helped in writing unit tests.